convert2rhel/main.py

```diff
@@ -180,7 +180,7 @@ def main_locked():
         _raise_for_skipped_failures(post_conversion_results)
         report.post_conversion_report(
             results=post_conversion_results,
-            include_all_reports=True,
```

Comment:
This feels unrelated

Reply:
Must've been missed in the rebase, i'll revert